src/basics.md

```diff
@@ -470,6 +471,80 @@ fn main() {
 }
 ```
 
+[ex-phone]: #ex-phone
+<a name="ex-phone"></a>
+## Extract phone numbers from text
+
+Attempts to process a static string as a list including US phone numbers.  The
+expression is checking that there are 7 digits that come in groups of 3 3 4,
```

Comment:
I would not try to explain the actual regex here. we are interested in the crate and its methods and don't try to teach regular expressions